benchmarks/gpu_pyfixest_errors.ipynb

```diff
@@ -0,0 +1,1478 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #45.                "ln_y": np.log(y + 1),
</pre><p>there is <code>np.log1p</code> (<a href="https://numpy.org/doc/stable/reference/generated/numpy.log1p.html" target="_blank">https://numpy.org/doc/stable/reference/generated/numpy.log1p.html</a>) hehe </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/py-econometrics/pyfixest/pull/784/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>